webpack.config.js

```diff
@@ -7,7 +7,7 @@ module.exports = (env, argv) => {
   const { mode } = argv
 
   return {
-    entry: './src/index.js',
+    entry: ['babel-polyfill', './src/index.js'],
```

Comment:
@mhiley , could you go into more detail about why this was added in?  Was it to get `async` to work?  There might be some downstream complications to worry about, but I'll have to investigate those and see what impact this has.  Sometimes babel has complained when there are two babel-polyfill's loaded on the page.  For example, I think I tried using async with [geoblaze](https://github.com/Geotiff/geoblaze) and georaster and ran into some complications.  This is more on me though, I need to do more research and figure out how to get async working on both projects.  Just alerting you now in case it comes up again.

Reply:
That's right, I added babel-polyfill to get `async` to work. I haven't tried testing the issue you mentioned with geoblaze yet, but it seems the implication is that georaster/geoblaze shouldn't have babel-polyfill at all, and that should be left up to the top-level application (e.g. `georaster-layer-for-leaflet-example` in our case). I'm not sure what the best practice is with this.

Reply:
Good point.  Honestly, I'm not sure there is a clear best practice.  I'll look into this.

Reply:
Apologies for the late reply.  I just moved to a new city for a new job and life's been a bit hectic.  I'm going to resolve this issue.  I'm going to switch the geoblaze code over to `require('georaster/src/index.js')` and I think we should be good.  It also wouldn't be a bad idea to add something to the README.md about this, but it's not a requirement for me to merge this PR.  Thanks for your patience :-)